.travis.yml

```diff
@@ -3,25 +3,14 @@ sudo: false
 language: node_js
 
 node_js:
-  - "0.8"
```

Comment:
Let's not drop these older versions please. This refactor shouldn't be dropping support for any Node.js versions.